src/daily-auths-report.tsx

```diff
@@ -36,18 +22,33 @@ interface Result {
    */
   iaa?: string;
 
+  // eslint-disable-next-line camelcase
   friendly_name: string;
```

Comment:
We could maybe consider mapping this back to a camel case property as part of `process`. Didn't really want to deal with it here.